File: C & C++ PRACTICE CODES/Nested for Pattern 4/code1.c

#include <stdio.h>

void main(){

	int row;
	printf("enter no. rows : ");
	scanf("%d",&row);

	for(int i = 1 ; i <= row ; i++){

		for(int j = 0; j < row ; j++ ){

			printf("\t%d\t",j + i);

		}
		printf("\n\n");

	}

}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/10/code1.java
import java.util.Scanner;

class TriangleDemo{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);
		System.out.print("Enter no of rows : ");
		int row = s.nextInt();

		int temp =1;


		for(int i = 0; i < row ; i++){

			for(int j = 0 ; j <= i ; j++ ){

				System.out.print(temp+"		");
				temp++;
			}

				System.out.print("\n\n");

		}

	}

}
